3.0.4/postgres/Dockerfile

```diff
@@ -12,3 +12,5 @@ COPY ./jdbc.properties $CATALINA_HOME/webapps/geonetwork/WEB-INF/config-db/jdbc.
 #Initializing database & connection string for GN
 COPY ./docker-entrypoint.sh /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
+
+CMD ["catalina.sh", "run"]
```

Comment:
Do you need to explicitly add this CMD when it is already declared on the base image (geonetwork:3.0.4)?